### PR TITLE
[PT-BR] Add `Beatmap_description`

### PR DESCRIPTION
**Pull request by GranularBow8193**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Walavouchey** [comment]
if you have someone else who can review this, tell them to comment here on github. otherwise leave the box unticked ty

**Comment by GranularBow8193** [comment]
@LeoFLT Can you review the pull requests that are open? My goal is to translate all wiki posts into PT-BR, so I would appreciate your help :)

File: wiki/Beatmap/Beatmap_description/pt-br.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Descrição do Beatmap
+
+**A descrição de Beatmap**  é uma área pequena embaixo da página do [beatmap](/wiki/Beatmap) que dá aos jogadores uma informação sobre o beatmap usando o [BBCode](/wiki/BBCode). Ele normalmente destaca fatos  sobre o criador do beatmap que ele acha suficientemente importantes, ou é usado para divulgar outros projetos de mapeamento.
```
Comment (LeoFLT) [inline wiki/Beatmap/Beatmap_description/pt-br.md:3]:
```suggestion
**A descrição de Beatmap** é uma área pequena embaixo da página do [beatmap](/wiki/Beatmap) que dá aos jogadores uma informação sobre o beatmap usando o [BBCode](/wiki/BBCode). Ele normalmente destaca fatos sobre o criador do beatmap que ele acha suficientemente importantes, ou é usado para divulgar outros projetos de mapeamento.
```

**Review by LeoFLT — CHANGES_REQUESTED** [review]
Looks good to me otherwise

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**